site/js/modules/webui/render.js

```diff
@@ -163,6 +237,22 @@ function submissionQuestions(questions) {
     return questionsHTML.join('');
 }
 
+function listCourseUsers(users) {
+    let messages = [];
+    for (const user of users) {
+        let userParts = [];
+
+        userParts.push(`Email: ${user['email']}`);
+        userParts.push(`Name: ${user['name']}`);
+        userParts.push(`Role: ${user['role']}`);
+        userParts.push(`Email: ${user['email']}`);
```

Comment:
Consider doing this in one literal statement:
```
let userParts = [
    `Email: ...`
    ...
]
```

You don't have to, but I would see if it looks nicer.